user-service/src/main/java/org/java/mentorship/user/repository/mapper/SessionMapper.java

```diff
@@ -0,0 +1,29 @@
+package org.java.mentorship.user.repository.mapper;
+
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Options;
+import org.apache.ibatis.annotations.Select;
+import org.java.mentorship.contracts.user.dto.Session;
+
+import java.util.List;
+import java.util.Optional;
+
+@Mapper
+public interface SessionMapper {
+    @Select("SELECT * FROM sessions")
+    List<Session> findAll();
+
+    @Insert("INSERT INTO sessions(session_key, expires_at, user_id) VALUES (#{sessionKey}, #{expiresAt}, #{userId})")
+    @Options(useGeneratedKeys = true, keyColumn = "id")
+    void insertSession(Session session);
```

Comment:
I'd create a convention around CRUD repositories / services

Methods should not include the entity in their name
It's obvious that the SessionMappe inserts a session